test/integration/api.bats

```diff
@@ -116,9 +116,29 @@ function teardown() {
 	[[ "${lines[*]}" ==  *"diff.txt"* ]]
 }
 
-# FIXME
 @test "docker events" {
-	skip
+	TEMP_FILE=$(mktemp)
```

Comment:
Yes. @jimmyxian can you start the container on a specific node and check the that right node is inside the event ?